sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryItemValidation/MetricsTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using Azure.Monitor.OpenTelemetry.Exporter.Tests.CommonTestFramework;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests.TelemetryItemValidation
+{
+    /// <summary>
+    /// The purpose of these tests is to validate the <see cref="TelemetryItem"/> that is created
+    /// based on interacting with <see cref="MeterProvider"/> and <see cref="Meter"/>.
+    /// </summary>
+    public class MetricsTests
```

Comment:
Not for this PR. we should add tests for other instruments too as a follow up.

Reply:
completely agree!